app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -94,7 +94,7 @@ class InstalledAddonDetailsFragment : Fragment() {
         switch.setOnCheckedChangeListener { v, isChecked ->
             val addonManager = v.context.components.addonManager
             switch.isClickable = false
-            remove_add_on.isEnabled = false
+            view.remove_add_on.isEnabled = false
```

Comment:
Interesting. Does kotlin view binding not recognize/handle the differences here?

Reply:
It looks like the Kotlin synthetic view binding variable get null at some point when we are switching between screen and as this code gets called async the null pointer exception happens